.github/workflows/cpp_extra.yml

```diff
@@ -330,12 +332,22 @@ jobs:
           cd cpp/examples/minimal_build
           ../minimal_build.build/arrow-example
 
+  odbc:
+    needs: check-labels
+    if: >-
+      needs.check-labels.outputs.force == 'true' ||
+      contains(fromJSON(needs.check-labels.outputs.ci-extra-labels || '[]'), 'CI: Extra') ||
+      contains(fromJSON(needs.check-labels.outputs.ci-extra-labels || '[]'), 'CI: Extra: C++')
+    name: C++ ODBC
+    uses: ./.github/workflows/cpp_odbc.yml
+
   report-extra-cpp:
     if: github.event_name == 'schedule' && always()
     needs:
       - docker
       - jni-linux
       - jni-macos
       - msvc-arm64
+      - odbc
```

Comment:
Could you keep this list in alphabetical order?

Reply:
Sure, I think the list is alphabetically ordered. Let me know if I missed something